src-java/stats-topology/stats-storm-topology/src/main/java/org/openkilda/wfm/topology/stats/StatsTopology.java

```diff
@@ -139,23 +146,30 @@ public StormTopology createTopology() {
         declareBolt(builder,
                 new TickBolt(topologyConfig.getStatisticsRequestInterval()), TICK_BOLT.name());
 
-        declareBolt(builder, new StatsRequesterBolt(persistenceManager), STATS_REQUESTER_BOLT.name())
-                .shuffleGrouping(TICK_BOLT.name());
+        declareBolt(builder, new StatsRequesterBolt(persistenceManager, ZooKeeperSpout.SPOUT_ID),
+                STATS_REQUESTER_BOLT.name())
+                .shuffleGrouping(TICK_BOLT.name())
+                .allGrouping(ZooKeeperSpout.SPOUT_ID);
 
-        declareBolt(builder, buildKafkaBolt(topologyConfig.getSpeakerTopic()), STATS_KILDA_SPEAKER_BOLT.name())
+        declareBolt(builder, buildKafkaBolt(topologyConfig.getSpeakerTopic(), getZkTopoName(),
+                getConfig().getBlueGreenMode()), STATS_KILDA_SPEAKER_BOLT.name())
```

Comment:
Does it make sense to introduce a dedicated "build" method instead of repeating "getZkTopoName(), getConfig().getBlueGreenMode()" each time?